documentation/src/main/asciidoc/userguide/chapters/fetching/Fetching.adoc

```diff
@@ -447,12 +452,11 @@ Besides the `FetchType.LAZY` or `FetchType.EAGER` JPA annotations,
 you can also use the Hibernate-specific `@Fetch` annotation that accepts one of the following ``FetchMode``s:
 
 SELECT::
-	The association is going to be fetched lazily using a secondary select for each individual entity,
-	collection, or join load.
-	It's equivalent to JPA `FetchType.LAZY` fetching strategy.
+	The association is going to be fetched using a secondary select for each individual entity,
+	collection, or join load. This mode can be used for either `FetchType.EAGER` or `FetchType.LAZY`.
```

Comment:
obviously the original content is wrong for `SELECT` is mostly used for `EAGER` loading, not restrictive to `LAZY` loading.